charts/nextjs/README.md

```diff
@@ -48,6 +48,18 @@ The following table lists the configurable parameters of the NextJS chart and th
 | `developmentImage.tag` | Development image tag for helm tests | `Chart.AppVersion` |
 | `developmentImage.pullPolicy` | Development image pull policy for helm tests | `IfNotPresent` |
 
+### E2E Tests Configuration
+
+| Parameter | Description | Default |
+|-----------|-------------|---------|
+| `e2eTests.image.repository` | E2E test image repository | `""` (required when using E2E tests) |
+| `e2eTests.image.tag` | E2E test image tag | `Chart.AppVersion` |
+| `e2eTests.image.pullPolicy` | E2E test image pull policy | `IfNotPresent` |
+| `e2eTests.smokeOnly` | Run only smoke tests | `true` |
+| `e2eTests.testFiles` | Custom test files to run | `[]` |
+| `e2eTests.env` | Additional environment variables for E2E tests | `[]` |
+| `e2eTests.resources` | Resource limits for E2E test pod | See values.yaml |
```

Comment:
Do not introduce a new image the development image contains everything needed to run the e2e tests